README.md

```diff
@@ -34,8 +34,8 @@ Come find us on the [development community chat](https://zulip.com/development-c
 
 - **Contributing code**. Check out our [guide for new
   contributors](https://zulip.readthedocs.io/en/latest/contributing/contributing.html)
-  to get started. We have invested into making Zulip’s code uniquely readable,
-  well tested, and easy to modify. Beyond that, we have written an extraordinary
+  to get started. We have invested in making Zulip’s code uniquely readable,
+  well-tested, and easy to modify. Beyond that, we have written an extraordinary
```

Comment:
This change is great but I'll squash in a further edit to improve the text in this paragraph.